### PR TITLE
Create 2019-12-10-Gamified-Chaos-Engineering-Tool-for-Kubernetes.md

### PR DESCRIPTION
**Pull request by lucky-sideburn**
>^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
> Remember to delete this note before submitting your pull request.
>
> For pull requests on 1.17 Features: set Milestone to 1.17 and Base Branch to dev-1.17
>
> For pull requests on Chinese localization, set Base Branch to release-1.16
> Feel free to ask questions in #kubernetes-docs-zh
>
> For pull requests on Korean Localization: set Base Branch to dev-1.16-ko.\<latest team milestone>
>
> If you need Help on editing and submitting pull requests, visit:
> https://kubernetes.io/docs/contribute/start/#improve-existing-content.
>
> If you need Help on choosing which branch to use, visit:
> https://kubernetes.io/docs/contribute/start#choose-which-git-branch-to-use.
>^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
>

### CODE REVIEW

**Comment by lucky-sideburn** [comment]
Hi!
I signed CLA :D

**Comment by lucky-sideburn** [comment]
Hi! Is ts ok my CLA?

**Comment by lucky-sideburn** [comment]
cla/linuxfoundation ok!

File: content/en/blog/_posts/2019-12-10-Gamified-Chaos-Engineering-Tool-for-Kubernetes.md [hunk 1]
```diff
@@ -0,0 +1,120 @@
+**KubeInvaders - Gamified Chaos Engineering Tool for Kubernetes**
```
Comment (kbarnard10) [inline content/en/blog/_posts/2019-12-10-Gamified-Chaos-Engineering-Tool-for-Kubernetes.md]: Remove bolded title and add front matter: 

---
layout: blog
title: "Gamified Chaos Engineering Tool for Kubernetes"
date: 2019-12-18
---

**Author:** Eugenio Marzo, SourceSense

**Comment by lucky-sideburn** [comment]
Hi! Please let me know I should do other changes to the pull request!

**Pull request merged**